docs-source/spring/content/setup/_index.md

```diff
@@ -121,3 +124,22 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
     <!-- spellchecker-disable -->
     {{< img name="oci-stack-apply-logs" size="large" lazy=false >}}
     <!-- spellchecker-enable -->
+
+## Setup the local workstation
+
+1. Install Local Tools
+
+    Before you set up the [Kubernetes access](./cluster-access), you must have already done the following:
```

Comment:
change the tense.  tell them what to do now, not what they needed to have already done